src/Services/AuthService.ts

```diff
@@ -0,0 +1,20 @@
+import * as jwt from "jsonwebtoken";
+import {ServiceError} from "./ServiceError";
+import {JwtPayload} from "../Types/JwtPayloadType";
+import {NodeService} from "./NodeService";
+import config from "../Config/Config";
+
+export class AuthService {
+
+    public async authorizerUser(token: string) {
```

Comment:
How will this function be reused? I don't see a way as it returns all user nodes but it's also called `authorizerUser`. Split the functionality. I think you don't need this auth service, you can do fine with existing services (Node and User).